tests/Doctrine/Tests/Common/Annotations/AbstractReaderTest.php

```diff
@@ -387,8 +387,11 @@ public function testInvalidAnnotationUsageButIgnoredClass()
         $reader = $this->getReader();
         $ref = new \ReflectionClass('Doctrine\Tests\Common\Annotations\Fixtures\InvalidAnnotationUsageButIgnoredClass');
         $annots = $reader->getClassAnnotations($ref);
-
-        $this->assertEquals(2, count($annots));
+        if ($annots[0] instanceof IgnoreAnnotation) {
+            $this->assertEquals(2, count($annots));
```

Comment:
`grep -r "assertCount" tests/|wc -l` = 13
 `grep -r "assertEquals" tests/|grep "count("|wc -l` = 51
This codebase clearly prefers assertEquals over assertCount.